experimental/dds/tree2/src/core/rebase/utils.ts

```diff
@@ -58,8 +73,10 @@ export interface RebasedCommits<TChange> {
  * @param changeRebaser - the change rebaser responsible for rebasing the changes in the commits of each branch
  * @param sourceHead - the head of the source branch, which will be rebased onto `targetHead`
  * @param targetHead - the commit to rebase the source branch onto
- * @returns the head of a rebased source branch, the cumulative change to the source branch (undefined if no change occurred),
- * and details about how the commits on the source branch changed
+ * @returns a tuple containing:
```

Comment:
Don't forget to update doc here. It doesn't need to talk about the thunkiness anymore, since that's now an implementation detail.